src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HearingsUpdateHearingRequest.java

```diff
@@ -76,7 +76,13 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
     }
 
     private boolean hasNoHearings(AsylumCase asylumCase) {
-        return asylumCase.read(CHANGE_HEARINGS, DynamicList.class).get().getListItems().isEmpty();
+        Optional<DynamicList> hearings = asylumCase.read(CHANGE_HEARINGS, DynamicList.class);
```

Comment:
I think you can do it in a more elegant way like this:

`return asylumCase.read(CHANGE_HEARINGS, DynamicList.class).map( changeHearings -> 
            changeHearings.getListItems().isEmpty()
        ).orElse(true);
`